ECHO-Sunrise.ipynb

```diff
@@ -232,62 +143,62 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Read in and map geojson for the selected geography\n",
-    "geo = region_field[select_region_widget.value][\"field\"].lower()\n",
-    "geo_json_data = geopandas.read_file(\"ECHO-Geo/ma_\"+geo+\".geojson\")\n",
+    "# Get the EPA data from the Stonybrook University database\n",
+    "echo_data_sql = \"select * from ECHO_EXPORTER where FAC_STATE = 'MA' and FAC_ACTIVE_FLAG='Y' \" \n",
+    "echo_data = get_data( echo_data_sql, 'REGISTRY_ID' )\n",
+    "num_facilities = echo_data.shape[0]\n",
     "\n",
-    "# Get rid of any null geographies\n",
-    "for geom in geo_json_data.iterrows():\n",
-    "    if(geom[1][\"geometry\"]==None):\n",
-    "        geo_json_data=geo_json_data.drop(geom[0])\n",
-    "        \n",
-    "m = folium.Map(\n",
-    "    #tiles='Mapbox Bright',\n",
-    ")\n",
-    "folium.GeoJson(\n",
-    "    geo_json_data,\n",
-    ").add_to(m)\n",
+    "# get selected district\n",
+    "geo_json_data = cd_data[(cd_data[\"District\"] == select_region_widget.value)] #where ids match\n",
     "\n",
-    "bounds = m.get_bounds()\n",
-    "m.fit_bounds(bounds)\n",
+    "# Join the facilities and the chosen district\n",
+    "this_district_data = echo_data.loc[(echo_data[\"FAC_DERIVED_CD113\"]==float(select_region_widget.value.strip(\"Congressional District\")))]\n",
     "\n",
-    "m"
+    "# Summarize \n",
+    "display(HTML(\"<h3>There are %s facilities in Massachussets currently tracked in the ECHO database.\" %(num_facilities)))\n",
+    "display(HTML(\"<h3>There are \"+ str(this_district_data.shape[0]) + \" facilities in this district.\"))\n",
+    "display(HTML(\"<h3>There are \"+ str(this_district_data.loc[this_district_data[\"RCRA_FLAG\"] == \"Y\"].shape[0]) + \" facilities regulated under RCRA in this district.</h3>\"))\n",
+    "display(HTML(\"<h3>There are \"+ str(this_district_data.loc[this_district_data[\"NPDES_FLAG\"] == \"Y\"].shape[0]) + \" facilities regulated under CWA in this district.</h3>\"))\n",
+    "display(HTML(\"<h3>There are \"+ str(this_district_data.loc[this_district_data[\"AIR_FLAG\"] == \"Y\"].shape[0]) + \" facilities regulated under CAA in this district.</h3>\"))\n",
+    "display(HTML(\"<h3>There are \"+ str(this_district_data.loc[this_district_data[\"GHG_FLAG\"] == \"Y\"].shape[0]) + \" facilities reporting GHGs in this district.</h3>\"))"
```

Comment:
To what extent is this what the notebook does vs. what people are encouraged to figure out? I think, for good reasons, the default is the former, but I also have reservations (because it gets complicated pretty quickly!)